app/views/follower_accounts/index.html.haml

```diff
@@ -6,4 +6,4 @@
 
 = render 'accounts/header', account: @account
 
-= render 'accounts/follow_grid', accounts: @accounts
+= render 'accounts/follow_grid', follows: @follows, accounts: @follows.map(&:account)
```

Comment:
I think instead of passing `accounts` into the partial, we can change the partial to check for `follows.empty?` and then have it run the `map` on follows before it passes along again to `grid_card`?